--- old/app/config/gimpguiconfig.c
+++ new/app/config/gimpguiconfig.c
@@ -321,7 +321,7 @@
                             "toolbox-wilber",
                             "Show toolbox wilber",
                             TOOLBOX_WILBER_BLURB,
-                            TRUE,
+                            FALSE,
                             GIMP_PARAM_STATIC_STRINGS);
 
   GIMP_CONFIG_PROP_BOOLEAN (object_class, PROP_TOOLBOX_GROUPS,
