Only in b: .configure.un~
--- a/configure
+++ b/configure
@@ -24152,27 +24152,6 @@
 
 CFLAGS="$gimp_save_CFLAGS"
 LIBS="$gimp_save_LIBS"
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $have_glib_networking" >&5
-$as_echo "$have_glib_networking" >&6; }
-
-if test "x$have_glib_networking" = "xno"; then
-
-  { $as_echo "$as_me:${as_lineno-$LINENO}: Eeeeeeeeeeeeeeeeeeeeek! Missing dep: glib-networking" >&5
-$as_echo "$as_me: Eeeeeeeeeeeeeeeeeeeeek! Missing dep: glib-networking" >&6;}
-  required_deps="$required_deps
-  - Error: missing dependency glib-networking"
-
-  required_deps="$required_deps
-      *** Test for glib-networking failed. This is required."
-elif test "x$have_glib_networking" != "xyes"; then
-  warning_glib_networking="
-WARNING: Test for glib-networking cannot be performed while cross-compiling.
-         Make sure glib-networking is installed, otherwise GIMP will not be able
-         to display the remote help pages through the help browser, nor will it
-         be able to open remote HTTPS (or other protocol using SSL/TLS) files.
-         HTTPS is becoming the expected standard and should not be considered
-         optional anymore."
-fi
 
 # AM_PATH_GLIB_2_0 already fills in the variable GLIB_COMPILE_RESOURCES.
 # Unfortunately it looks it up using $PKG_CONFIG, which search in
Only in b: configure~
