--- old/data/meson.build
+++ new/data/meson.build
@@ -9,14 +9,14 @@
 custom_target('org.pitivi.Pitivi.desktop',
     output : 'org.pitivi.Pitivi.desktop',
     input : 'org.pitivi.Pitivi.desktop.in',
-    command : [intltool_merge, '--desktop-style', podir, '@INPUT@', '@OUTPUT@'],
+    command : ['cp', '@INPUT@', '@OUTPUT@'],
     install : true,
     install_dir : join_paths(get_option('datadir'), 'applications'))
 
 custom_target('org.pitivi.Pitivi.appdata.xml',
     output : 'org.pitivi.Pitivi.appdata.xml',
     input : 'org.pitivi.Pitivi.appdata.xml.in',
-    command : [intltool_merge, '--xml-style', podir, '@INPUT@', '@OUTPUT@'],
+    command : ['cp', '@INPUT@', '@OUTPUT@'],
     install : true,
     install_dir : join_paths(get_option('datadir'), 'appdata'))
 
--- old/meson.build
+++ new/meson.build
@@ -1,9 +1,7 @@
 project('pitivi', 'c', version : '0.999', meson_version : '>= 0.41.0')
 
 python = find_program('python3')
-intltool_merge = find_program('intltool-merge')
 itstool = find_program('itstool')
-msgfmt = find_program('msgfmt')
 
 python_dep = dependency('python3', version : '>= 3.3')
 
